### DISCUSSION TITLE
Im getting an error.

### DISCUSSION THREAD

**Discussion by Chayni** [UltimaHoarder/UltimaScraper#1368] [Q&A] [answered]
Hey, I dont know shit about python so I tried as hard to try to follow the tutorial.
After I finished setting up the auth.json and I try running the app I get this error, any idea why?
![a5b39ba7b8375043b4cef5a517fa27aa](https://user-images.githubusercontent.com/22572495/122604056-013a8080-d07e-11eb-992f-b4a0b9ab8fd7.png)

**Comment by ghost** [comment, marked as answer]
Looks like you did not run:

pip install -r requirements.txt

**Reply by Chayni** [reply]
I actually did, how do I check which were installed and which are missing?

**Reply by ghost** [reply]
Just run it again it will say:

Requirement already satisfied: *package-name*
